### PR TITLE
Autogenerated/wargaming.worldoftanks/86cc2deb 3ce1 416f 8d75 f687884abec0

### PR DESCRIPTION
**Pull request by mddtest**
- [X ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [X ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16044)

### CODE REVIEW

**Comment by KaranKad** [comment]
Making `w` from `wargaming` capital would have been appreciated.

File: manifests/w/wargaming/GameCenter/21.03.00.5224/wargaming.GameCenter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate
+
+PackageIdentifier: Wargaming.GameCenter
+PackageVersion: 21.03.00.5224
```
Comment (ItzLevvie) [inline manifests/w/wargaming/GameCenter/21.03.00.5224/wargaming.GameCenter.installer.yaml:4]:
```suggestion
PackageVersion: 21.3.0.5224
```
image

File: manifests/w/wargaming/GameCenter/21.03.00.5224/wargaming.GameCenter.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Wargaming.GameCenter
+PackageVersion: 21.03.00.5224
```
Comment (ItzLevvie) [inline manifests/w/wargaming/GameCenter/21.03.00.5224/wargaming.GameCenter.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 21.3.0.5224
```

File: manifests/w/wargaming/GameCenter/21.03.00.5224/wargaming.GameCenter.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Wargaming.GameCenter
+PackageVersion: 21.03.00.5224
```
Comment (ItzLevvie) [inline manifests/w/wargaming/GameCenter/21.03.00.5224/wargaming.GameCenter.yaml:4]:
```suggestion
PackageVersion: 21.3.0.5224
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/w/wargaming/GameCenter/21.03.00.5224/wargaming.GameCenter.yaml (hunk 3 above)
Comment (mddtest) [reply manifests/w/wargaming/GameCenter/21.03.00.5224/wargaming.GameCenter.yaml:4]: Resubmitted with correct folder structure and updated package version.

**Pull request closed without merging**